docs/OVERVIEW.md

```diff
@@ -0,0 +1,453 @@
+for more information.
+
+### Cloud IO Support
+
+[File-System Specification (fsspec)](https://github.com/intake/filesystem_spec) allows
+for common object storage services (S3, GCS, etc.) to act like normal filesystems by
+following the same base specification across them all. BioIO utilizes this
+standard specification to make it possible to read directly from remote resources when
+the specification is installed.
+
+```python
+from bioio import BioImage
+
+# Get a BioImage object
+img = BioImage("http://my-website.com/my_file.tiff")
+img = BioImage("s3://my-bucket/my_file.tiff")
+img = BioImage("gcs://my-bucket/my_file.tiff")
+
+# Or read with specific filesystem creation arguments
+img = BioImage("s3://my-bucket/my_file.tiff", fs_kwargs=dict(anon=True))
+img = BioImage("gcs://my-bucket/my_file.tiff", fs_kwargs=dict(anon=True))
+
+# All other normal operations work just fine
+```
+
+Remote reading requires that the file-system specification implementation for the
+target backend is installed.
+
+-   For `s3`: `pip install s3fs`
+-   For `gs`: `pip install gcsfs`
+
+See the [list of known implementations](https://filesystem-spec.readthedocs.io/en/latest/?badge=latest#implementations).
+
+### Saving to OME-TIFF
+
+The simpliest method to save your image as an OME-TIFF file with key pieces of
+metadata is to use the `save` function.
+
+```python
+from bioio import BioImage
+
+BioImage("my_file.czi").save("my_file.ome.tiff")
+```
+
+**Note:** By default `BioImage` will generate only a portion of metadata to pass
+along from the reader to the OME model. This function currently does not do a full
+metadata translation.
+
+For finer grain customization of the metadata, scenes, or if you want to save an array
+as an OME-TIFF, the writer class can also be used to customize as needed.
+
+```python
+import numpy as np
+from bioio.writers import OmeTiffWriter
+
+image = np.random.rand(10, 3, 1024, 2048)
+OmeTiffWriter.save(image, "file.ome.tif", dim_order="ZCYX")
+```
+
+See
+[OmeTiffWriter documentation](https://bioio-devs.github.io/bioio/bioio.writers.html)
+for more details.
+
+#### Other Writers
+
+In most cases, `BioImage.save` is usually a good default but there are other image
+writers available. For more information, please refer to
+[our writers documentation](https://bioio-devs.github.io/bioio/bioio.writers.html).
+
+## Development
+
+See our
+[developer resources](https://bioio-devs.github.io/bioio/developer_resources.html)
+for information related to developing the code.
+
+## Citation
+
+If you find `bioio` useful, please cite this repository as:
+
+> Eva Maxfield Brown, Dan Toloudis, Jamie Sherman, Madison Swain-Bowden, Talley Lambert, Sean Meharry, Brian Whitney, AICSImageIO Contributors (2023). BioIO: Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python [Computer software]. GitHub. https://github.com/bioio-devs/bioio
```

Comment:
still not sure if we need to change the contributors list or just completely preserve the entire AICSImageIO lineage.  It still says `AICSImageIO Contributors` and maybe we should either add `BioIO Contributors` or replace.  Also kinda wondering if this conversation needs to involve AICS governance.